apps/prairielearn/src/lib/client/safe-db-types.ts

```diff
@@ -8,27 +8,45 @@ import {
 
 /** Courses */
 
-export const RawStaffCourseSchema = RawCourseSchema.omit({
-  yearly_enrollment_limit: true,
-  sharing_token: true,
-});
-export const StaffCourseSchema = RawStaffCourseSchema.brand<'StaffCourse'>();
-export type StaffCourse = z.infer<typeof StaffCourseSchema>;
-
-export const RawStudentCourseSchema = RawStaffCourseSchema.omit({
+export const RawStaffCourseSchema = RawCourseSchema.pick({
```

Comment:
Not that I mind either way, but is there a reason that we are exporting the Raw versions of these schemas?